services/apps/src/apps/metadata/reference_genomes.clj

```diff
@@ -83,18 +91,33 @@
     (sql/update genome_reference (set-fields update-values) (where {:id reference-genome-id}))
```

Comment:
I think that this function (`update-reference-genome`) should also validate that, if a new name or path is provided, it does not conflict with anything.